### ISSUE TITLE
default PAGER pager does not exist

### ISSUE THREAD

**Issue by jayvdb** [jwilk/mwic#1]
When the `PAGER` env var is missing, mwic defaults to `pager`, which is not typically installed, which results in a non-intuitive exception when it is missing.

http://packages.ubuntu.com/search?searchon=contents&keywords=pager&mode=exactfilename&suite=xenial&arch=any
https://packages.debian.org/search?searchon=contents&keywords=pager&mode=path&suite=stable&arch=any
https://admin.fedoraproject.org/pkgdb/packages/?motif=%2Apager

`more` would be a more sensible default, however a pure python package would be better.

**Comment by jwilk** [comment]
FWIW, `pager` does exist on Debian and its derivatives:
https://www.debian.org/doc/debian-policy/ch-customized-programs.html#s11.4
(`manpages.ubuntu.com` and `packages.debian.org` are not aware of the alternatives mechanism, so they won't find it.)

You are right that `pager` is not very portable. We should fall back to `more` which is in POSIX.

I'll get it fixed soon.

**Comment by jwilk** [comment]
I implemented `more` fall-back in 9303b140a61c.

I don't like the idea of pure-Python pager, but if you feel strongly about it, feel free to open a separate issue.

**Comment by jayvdb** [comment]
No, I dont feel strongly about it.  `more` should be fine.
I note that `sensible-pager` (in `sensible-utils`) is available for Fedora 23 where it is

``` bash
#!/bin/sh

${PAGER:-pager} "$@"
ret="$?"
if [ "$ret" -eq 126 ] || [ "$ret" -eq 127 ]; then
more "$@"
ret="$?"
    if [ "$ret" -eq 126 ] || [ "$ret" -eq 127 ]; then
        echo "Couldn't find a pager!" 1>&2
        echo "Set the \$PAGER environment variable to your desired pager." 1>&2
        exit 1
    fi
fi
```

I doubt it is in OSX, after a quick look on the web, and even less likely to be on other Unix.  So skipping that and just using `more` seems appropriate.

**Comment by jwilk** [comment]
I've just released 0.5.1, which includes aforementioned fix.
